radius: 25.0
cy: 70.0
traj_length: 50
timer_period: 0.1
max_velocity: 10.0
mpc_steps: 4
mpc_dt: 0.25
kcte: 1.0
kepsi: 1.0
kev: 0.001